packages/board-server/tests/integration/board-api.test.ts

```diff
@@ -0,0 +1,489 @@
+import { suite, test } from 'node:test';
```

Comment:
Wow, this is great! Thank you for writing these.

Reply:
Seriously. The hardest part of tests is writing them in the first place. It's so much easier to modify tests that already exist than to create them from scratch. Thank you.